packages/remix-dev/cli/commands.ts

```diff
@@ -44,12 +44,15 @@ export async function init(
     });
   }
 
-  let initFn = require(initScript);
+  let initFn = await import(initScript);
+
   if (typeof initFn !== "function" && initFn.default) {
     initFn = initFn.default;
   }
   try {
-    await initFn({ packageManager, rootDirectory: projectDir });
+    if (typeof initFn === "function") {
```

Comment:
Covered by the "succeeds for a remix.init script that doesn't default export a function" test.